docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,177 +575,198 @@ The following features are not supported:
-
-```
-public int Shape
-{
-    get
-    {
-        return _shape;
-    }
-    set
-    {
-        _shape = value;
-
-        /// Create and register the primitive visual
-        PropertyMap primitiveVisual = new PropertyMap();
-        primitiveVisual.Add(Visual.Property.Type, new PropertyValue((int)Visual.Type.Primitive))
-                       .Add(PrimitiveVisualProperty.Shape, new PropertyValue(_shape))
-                       .Add(PrimitiveVisualProperty.BevelPercentage, new PropertyValue(0.3f))
-                       .Add(PrimitiveVisualProperty.BevelSmoothness, new PropertyValue(0.0f))
-                       .Add(PrimitiveVisualProperty.ScaleDimensions, new PropertyValue(new Vector3(1.0f, 1.0f, 0.3f)))
-                       .Add(PrimitiveVisualProperty.MixColor, new PropertyValue(new Vector4((245.0f/255.0f), (188.0f/255.0f), (73.0f/255.0f), 1.0f)));
-        _primitiveVisual = VisualFactory.Get().CreateVisual(primitiveVisual);
-        RegisterVisual(PrimitiveVisualPropertyIndex, _primitiveVisual);
-
-        /// Set the primitive visual depth index
-        _primitiveVisual.DepthIndex = PrimitiveVisualPropertyIndex;
-    }
-}
-```
-
-### Primitive Visual Examples
+The following example shows how to use a MeshVisual.
+The `.obj` and `.mtl` files and directory with textures used the same path as was used for the [ImageVisual](#imageVisual).
+   ```csharp
+   string _imageUrl = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
+
+   MeshVisual _meshVisual = new MeshVisual();
+
+   _meshVisual.ObjectURL    = _imageUrl + "Dino.obj";
+   _meshVisual.MaterialtURL = _imageUrl + "Dino.mtl";
+   _meshVisual.TexturesPath = _imageUrl + "textures/";
+
+   _visualView.AddVisual("Mesh-Image", _meshVisual);
+   ```
+
+**Figure: MeshVisual**
+
+![Mesh visual](media/visuals/MeshVisualExample.png)
+
+
+
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="primitiveVisual"></a>
+## PrimitiveVisual
+
+The PrimitiveVisual renders a simple three-dimensional shape, such as cube or sphere. 
+By default, shapes are generated with clockwise winding and back-face culling.
+
+
+The following table lists the supported properties: 
+
+**Table: PrimitiveVisual properties - all values are optional.**
+
+| Property            | Type                       | Shape to which the property applies  | Description                                                                                                        |
+|---------------------|----------------------------|--------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| `Shape`             | `PrimitiveVisualShapeType` | all                                  | The shape to render (see tab. [with possible values](#table_primit_possible_values) - by default is `Sphere`).     |
```

Comment:
```suggestion 
| `Shape`             | `PrimitiveVisualShapeType` | all                                  | The shape to render. For more information, see [possible values](#table_primit_possible_values) (default  `Sphere`).     |

Reply:
I am waiting for your reply to the https://github.com/Samsung/tizen-docs/pull/1367#discussion_r537940183 - I will use the same scheme everywhere.
currently I write like this:

The shape to render (default `Sphere`; see the [below table](#table_primit_possible_values) for more information).